config/routes.rb

```diff
@@ -50,6 +50,7 @@
         get "copy_prior_recommendations"
         post "complete"
         post "associate_granule_to_collection"
+        delete 'records/:id', to: 'records#destroy', as: :destroy_record
```

Comment:
you should don't need a route for this, rails implies certain routes, delete being to destroy

Reply:
Removed, good to know, thanks! 👍